### PR TITLE
Adding alerts to home page and refactoring beers.js and stats.js

### PR DESCRIPTION
**Pull request by EnriqueRivera**

### CODE REVIEW

File: src/main/resources/templates/index.html [hunk 1]
```diff
@@ -89,8 +89,17 @@ <h2>Consumed Beers</h2>
...
-                    <button type="submit" class="btn btn-primary">Guardar</button>
+                    <button class="btn btn-primary">Guardar</button>
                 </form>
+                
+                <div class="alert alert-success fade in alert-dismissible" style="margin-top:15px; display: none;">
+				    <strong>Success!</strong> Beer has been added.
+				</div>
+				
+				<div class="alert alert-danger fade in alert-dismissible" style="margin-top:15px; display: none;">
+				    <strong>Error </strong> adding beer info.
```
Comment (leogtzr) [inline src/main/resources/templates/index.html]: It should say "adding drink info".

**Review by leogtzr — COMMENTED** [review]
Little change about an error message.

**Comment by EnriqueRivera** [comment]
Done.

**Comment by leogtzr** [comment]
Thanks!

**Pull request merged**